src/stems/stem-tracker.js

```diff
@@ -21,49 +22,48 @@ function StemTracker(editor, onClick) {
 
   /** Listen for changes in content and update. */
   editor.addListener({
-    onContent: function () {
-      updateStems( this.editorElem );
+    onKeyup: function (e) {
+      updateStems();
       return false;
     },
+    onMouseup: function (e) {
+      updateStems();
+      return false
+    }
   });
 
 
-  function updateStems(editorElem){
-    var previousStems = stems;
-    stems = [];
-
-    /** Loops through all top level elements */
-    var topLevelElems = getTopLevelElems( topLevelTags );
-    for (var i=0; i < topLevelElems.length; i++){
-      var elem = topLevelElems[i];
-      var stem = Stem.getOrCreate( elem, onClick, me.containerElem );
-      stems.push(stem);
-    }
-
-    /** Removes orphaned stems */
-    for (var i=0; i < previousStems.length; i++){
-      var previousStem = previousStems[i];
-      if (stems.indexOf(previousStem) < 0){
-        previousStem.remove();
+  function getTopLevelBlockElem(currentNode){
+    for (var node = currentNode; node != null; node = node.parentNode){
+      if (util.isElement(node) && util.isBlock(node)) {
+        return node;
       }
     }
-  }
+    return false;
+  } 
+
+  function updateStems(){
+    var point = editor.selection().getRange().focus;
+    var previousStem = stem;
+    if (stem){
+      console.log("remove?");
```

Comment:
cleanup debug logging

my trick is to always use console.debug, and then it's easy to grep for